node/Dockerfile

```diff
@@ -25,7 +25,7 @@ ADD ./node .
 RUN if [ "$network" != "devnet" ]; then apt-get update && apt-get install libcap2-bin -y; fi
 
 RUN --mount=type=cache,target=/root/.cache --mount=type=cache,target=/go \
-  go build -race -gcflags="all=-N -l" --ldflags '-extldflags "-Wl,--allow-multiple-definition"' -mod=readonly -o /guardiand github.com/alephium/wormhole-fork/node
+  go build -gcflags="all=-N -l" --ldflags '-extldflags "-Wl,--allow-multiple-definition"' -mod=readonly -o /guardiand github.com/alephium/wormhole-fork/node
```

Comment:
The purpose of disabling the race detector is to solve the OOM(out of memory) error. Currently, the guardian enables the race detector at runtime to detect memory read-write conflicts. And I did some tests on my local side:

1. If we enable the race detector and lock memory, it leads to an OOM error.
2. If we only enable the race detector but without locking memory, the memory usage is relatively stable, around 300-400MB.
3. If we only lock memory but disable the race detector, the memory usage is also relatively stable, but much higher than case 2, around 2.5GB.

I checked the latest wormhole code, I found that they made the `-race` flag [optional](https://github.com/wormhole-foundation/wormhole/blob/21276f7f752b2f538bef49ace3046d7084bfe641/node/Dockerfile#L23) and no longer [lock memory on the testnet](https://github.com/wormhole-foundation/wormhole/blob/21276f7f752b2f538bef49ace3046d7084bfe641/node/cmd/guardiand/node.go#L419).

Additionally, [some issues](https://github.com/golang/go/issues/37233) suggest that the golang race detector may have memory leaks.

And OOM would not happen again after deploying the new code to testnet.